Resources/config/doctrine/Subscription.orm.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+  <entity name="Integrated\Bundle\SubscriptionBundle\Entity\Subscription" table="subscription">
```

Comment:
more white lines makes it more readable, I would suggest a new white line after every defines ORM variable